### PR TITLE
Styled section header to fit the new cell layout.

### PR DESCRIPTION
**Pull request by jonasjensen-brolstark**
@PetarRing and @SilverGull would you look at this and check that i have not messed anything up in your cell implementation?

### CODE REVIEW

File: src/kirby/components/list/list.component.scss [hunk 1]
```diff
@@ -5,19 +5,37 @@ ul {
   margin-top: 0;
   font-size: font-size('s');
   margin-block-end: 0;
+
   li {
-    background-color: white;
     background-color: var(--kirby-surface);
     border-bottom: 1px solid var(--kirby-background);
     padding: 16px;
```
Comment (PetarRing) [inline src/kirby/components/list/list.component.scss]:
```suggestion
    padding:  size('m');
```

Comment (PetarRing) [reply src/kirby/components/list/list.component.scss]: I think this was left out when Peter merged my branch onto his 😄

Comment (PetarRing) [reply src/kirby/components/list/list.component.scss]: The hover color could use some transparancy, hence the thing I tried with rgba 😄

File: src/kirby/components/list/list.component.tns.scss [hunk 2]
```diff
@@ -1,11 +1,8 @@
 @import "../../scss/native-imports";
-$list-item-height: 44;
-$list-item-padding: 16;
-$list-header-height: 38;
 .row {
   padding: 16 8 16 8;
   justify-content: space-between;
   flex-direction: row;
-  border-color: #d6d6d6;
+  border-color: #f5f5f5;
```
Comment (PetarRing) [inline src/kirby/components/list/list.component.tns.scss]: Shouldn't this be a theming color? 
```suggestion
  border-color: get-color('kirby-background');
```

File: src/kirby/components/list/list.component.tns.scss [hunk 3]
```diff
@@ -1,11 +1,8 @@
 @import "../../scss/native-imports";
-$list-item-height: 44;
-$list-item-padding: 16;
-$list-header-height: 38;
 .row {
   padding: 16 8 16 8;
```
Comment (PetarRing) [inline src/kirby/components/list/list.component.tns.scss]: Should this be a theming size? 😄 I am not sure if you need to specify all padding or only need left/right and top..
```suggestion
  padding: size('gutter') size('gutter-small') size('gutter') size('gutter-small');
```

File: src/kirby/components/list/list.component.scss (hunk 1 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/list/list.component.scss]: Should that not be a themed variable if it should have transparancy? or a lighter version of `--kirby-secondary` ?

File: src/kirby/components/list/list.component.tns.scss (hunk 3 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/list/list.component.tns.scss]: Thanks didn't know we had cutter sizes.

File: src/kirby/components/list/list.component.tns.scss (hunk 2 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/list/list.component.tns.scss]: Awesome was looking for how to use variables in native.

**Pull request merged**